src/gam.py

```diff
@@ -1368,24 +1368,26 @@ def addDelegates(users, i):
       systemErrorExit(2, u'%s is not a valid argument for "gam <users> delegate", expected to' % sys.argv[i])
     i += 1
   delegate = sys.argv[i].lower()
-  if not delegate.find(u'@') > 0:
+  atLoc = delegate.find(u'@')
+  if atLoc == -1:
     delegate_domain = GC_Values[GC_DOMAIN].lower()
     delegate_email = u'%s@%s' % (delegate, delegate_domain)
   else:
-    delegate_domain = delegate[delegate.find(u'@')+1:].lower()
+    delegate_domain = delegate[atLoc+1:].lower()
     delegate_email = delegate
   i = 0
   count = len(users)
   emailsettings = buildGAPIObject(u'email-settings')
   for delegator in users:
     i += 1
-    if delegator.find(u'@') > 0:
-      delegator_domain = delegator[delegator.find(u'@')+1:].lower()
-      delegator_email = delegator
-      delegator = delegator[:delegator.find(u'@')]
-    else:
+    atLoc = delegator.find(u'@')
```

Comment:
let's not use same atLoc variable for delgate and delegator, could cause issues down the road....